cmd/zed/lake/ztests/log.yaml

```diff
@@ -0,0 +1,33 @@
+script: |
+  export ZED_LAKE_ROOT=test
+  zed lake init -q
+  zed lake create -q -p POOL
```

Comment:
Existing tests almost always use a pool named test, and I think we should stick with that convention.

Reply:
Sure